app/grandchallenge/cases/image_builders/dicom.py

```diff
@@ -151,6 +151,7 @@ def _create_sitk_image(dcm_array):
 
 def _process_dicom_file(dicom_ds):  # noqa: C901
     ref_file = pydicom.dcmread(str(dicom_ds.headers[0]["file"]))
+    ref_origin = tuple(float(i) for i in ref_file.ImagePositionPatient)
```

Comment:
DICOM files are not guaranteed to have ImagePositionPatient

Reply:
Ah true, I forgot the if statement there